http.go

```diff
@@ -155,6 +155,15 @@ func (req *Request) SetConnectionClose() {
 	req.Header.SetConnectionClose()
 }
 
+// SetUseOriginalPath sets the useOriginalPath flag.
+// If this flag is set to true, path values are sent as-is without normalization
+//
+// By default path values are normalized, i.e.
+// extra slashes are removed, special characters are encoded.
+func (req *Request) SetUseOriginalPath(useOriginalPath bool) {
+	req.uri.useOriginalPath = useOriginalPath
+}
```

Comment:
I'm wondering if this function should be called `DisablePathNormalizing`. Just like the `DisableHeaderNamesNormalizing` that some of our APIs have.

I'm also wondering why this is a method of `Request`? Why not of `URI`? So you will have to do `Request.URI().DisablePathNormalizing()`?

Reply:
Sure, that sounds like a better name. Will change.

It doesn't matter to me whether the method is on `Request` or `URI`, I'll change it to be `URI` only.